ods_ci/tests/Resources/CLI/AcceleratorProfiles/AcceleratorProfiles.resource

```diff
@@ -0,0 +1,96 @@
+*** Settings ***
+Documentation    Collection of keywords to interact with the administration of Accelerator Profiles via CLI
+
+
+*** Variables ***
+${ACCELERATOR_RESOURCES_DIRPATH}=    ods_ci/tests/Resources/Files/accelerator
+
+
+*** Keywords ***
+Create An Accelerator Profile Via CLI
+    [Documentation]    Create an instance of Accelerator Profile using OC
+    [Arguments]   ${name}  ${ns}=redhat-ods-applications
+    ${rc}    ${out}    Run And Return Rc And Output   oc create -f ${ACCELERATOR_RESOURCES_DIRPATH}/${name}.yaml -n ${ns}
```

Comment:
@FedeAlonso Can you fix ` The assignment sign is not consistent within the file. Expected '{{ expected_sign }}' but got '{{ actual_sign }}' instead ` 
You probably need `${rc}    ${out}=` ...

Reply:
Fixed in 
822a36b